1496 recreate proofs (#1506)
MetaMask/metamask-docs: services/reference/optimism/index.md

:::info see also

- For more information on the Optimism network, please see the
  [official Optimism developer documentation](https://docs.optimism.io/).

> - Developers should be aware of:
>   - some differences between the [behavior of Ethereum and Optimism](https://docs.optimism.io/stack/differences)
>   - the JSON-RPC methods specific to [Optimism Bedrock](https://docs.optimism.io/builders/node-operators/json-rpc)

Reviewer: ```suggestion
- See the differences between the [behavior of Ethereum and Optimism](https://docs.optimism.io/stack/differences).
 - See the JSON-RPC methods specific to [Optimism Bedrock](https://docs.optimism.io/builders/node-operators/json-rpc).
```

Let's minimize nesting of lists where possible.
